lib/components/src/tabs/tabs.tsx

```diff
@@ -143,18 +166,7 @@ export const Tabs = React.memo(
   }
 );
 Tabs.displayName = 'Tabs';
-Tabs.propTypes = {
-  id: PropTypes.string,
-  children: PropTypes.node,
-  tools: PropTypes.node,
-  selected: PropTypes.string,
-  actions: PropTypes.shape({
-    onSelect: PropTypes.func.isRequired,
-  }).isRequired,
-  absolute: PropTypes.bool,
-  bordered: PropTypes.bool,
-};
-Tabs.defaultProps = {
+(Tabs as any).defaultProps = {
```

Comment:
can be set as a static property?

Reply:
Hmm I did not find an easy way to do it as `Tabs` is created using `React.memo`